goldilocks/src/lib.rs

```diff
@@ -484,17 +482,8 @@ mod tests {
     }
 
     #[test]
-    fn inverse() {
-        test_inverse::<Goldilocks>();
-    }
-
-    #[test]
-    fn two_adic_subgroup_zerofier() {
-        test_two_adic_subgroup_zerofier::<Goldilocks>();
-    }
-
-    #[test]
-    fn two_adic_coset_zerofier() {
-        test_two_adic_coset_zerofier::<Goldilocks>();
+    fn common_tests() {
+        field_test::<Goldilocks>();
+        two_adic_field_test::<Goldilocks>();
```

Comment:
nit: Same as for BabyBear, is there a reason you think batching these tests is better?